src/test/java/pl/wedding/management/service/reservation/domain/converter/ReservationConverterTest.java

```diff
@@ -0,0 +1,134 @@
+package pl.wedding.management.service.reservation.domain.converter;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pl.wedding.management.service.reservation.domain.entity.ReservationDto;
+import pl.wedding.management.service.reservation.domain.entity.ReservationEntity;
+import pl.wedding.management.service.reservation.domain.enums.Occasion;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ReservationConverterTest {
+
+    ReservationConverter reservationConverter;
+    ReservationDto reservationDto;
+
+    @BeforeEach
+    void setUp() {
+        reservationConverter = new ReservationConverter();
+        reservationDto = new ReservationDto(
+                1,
+                LocalDateTime.of(2020,Month.OCTOBER,3,18,30,0),
+                LocalDateTime.of(2020,Month.OCTOBER,4,6,30,0),
+                new Object(),
+                new Object(),
+                200,
+                Occasion.DIVORCE_PARTY
+        );
+    }
+
+    @Test
+    void convertFromEntityIsEqual() {
+
+        //given
+        ReservationEntity reservationEntity = new ReservationEntity(
+                1,
+                LocalDateTime.of(2020,Month.OCTOBER,3,18,30,0),
+                LocalDateTime.of(2020,Month.OCTOBER,4,6,30,0),
+                new Object(),
+                new Object(),
+                200,
+                Occasion.DIVORCE_PARTY
+        );
+
+        //when
+        ReservationDto reservationDtoFromConverter = reservationConverter.convertFromEntity(reservationEntity);
+
+        //then
+        assertEquals(reservationDto, reservationDtoFromConverter);
+    }
+
+    @Test
+    void convertFromEntityCutSecondsIsEqual() {
+
+        //given
+        ReservationEntity reservationEntity = new ReservationEntity(
+                1,
+                LocalDateTime.of(2020,Month.OCTOBER,3,18,30,20,20),
+                LocalDateTime.of(2020,Month.OCTOBER,4,6,30,20),
+                new Object(),
+                new Object(),
+                200,
+                Occasion.DIVORCE_PARTY
+        );
+
+        //when
+        ReservationDto reservationDtoFromConverter = reservationConverter.convertFromEntity(reservationEntity);
+
+        //then
+        assertEquals(reservationDto, reservationDtoFromConverter);
+    }
+
+    @Test
+    void convertFromEntityIsNotEqual() {
```

Comment:
W momencie kiedy chcesz przetestować tą samą logikę ale z innymi parametrami wejściowym to używaj DataProvidera. W ten sposób unikniesz powtarzania kodu. Doczytaj czym jest DataProvider, jak go używać i zmień proszę tą metodę - i wszystkie inne, do których będziesz mógł to zastosować.

Reply:
Ok, poczytałem trochę ale po refaktorze Equals() nie widzę tu miejsca na tego typu zmiany. Skoro w equals sprawdzany jest tylko Id, to nie testuję każdego innego parametru osobno.